src/components/UserCreatedBounties/UserBountyPage/BountySubmission/CreateSubmissionForm.tsx

```
@@ -0,0 +1,348 @@
+			setOpenModal(false);
+			if (onEditSuccess && isUsedForEditing && submission) {
+				onEditSuccess({
+					...submission,
+					title: values.title,
+					content: values.description,
+					reqAmount: values.requestAmount,
+					link: values.links || '',
+					createdAt: submission.createdAt ?? new Date()
+				});
+			} else {
+				fetchSubmissions && fetchSubmissions();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)